test/functional/install/install-multiple.bats

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bats
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME" 10
+	create_bundle -n test-bundle -f /file_1,/file_2 "$TEST_NAME"
+	sudo mkdir -p "$TARGETDIR"/usr/share/clear/bundles
+	sudo touch "$TARGETDIR"/usr/share/clear/bundles/test-bundle
+
+
+}
+
+@test "install multiple bundles" {
```

Comment:
I'm confused about the description of this test, it mentions installing multiple bundles but yet I see only one bundle being explicit here. The "multiple" bundles is because of the os-core bundle that comes by default?

Reply:
Never mind, I see now that it must be it. Maybe adding `assert_file_exists "$TARGETDIR"/core` to the list of verified files will make it easier to see that

Reply:
Correct.